### PR TITLE
feat: 外部页面 (#50)

### LINKED ISSUE #50: 外部页面
### 清晰简洁的问题描述

1. 内嵌页面
   - 项目文档
   - 组件库文档
2. 外链：
    演示打开菜单打开外链功能，项目文档

### 附加上下文

_No response_

### PR DESCRIPTION
**Pull request by wang01h2**
### What is the purpose of this pull request? 

- [ ] Bug fix
- [x] New Feature
- [ ] Documentation update
- [ ] Other

### Before submitting the PR, please make sure you do the following

- [x] Check that there isn't already a PR that solves the problem the same way to avoid creating a duplicate.
- [x] Provide a description in this PR that addresses **what** the PR is solving, or reference the issue that it solves (e.g. `fixes #123`).

### Views
add inline page

### Route
add inline route

close #50

### CODE REVIEW

**Comment by Hongbusi** [comment]
在描述中关联一下对应的 issue，格式：`close #issueId`。

**Comment by wang01h2** [comment]
fix done

**Comment by RainyNight9** [comment]
对，这个先不要合并，如果动态菜单那个 合并完了，这个需要重新在更改一下

File: src/views/page/inline/index.vue [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+<style lang="less" scoped>
+.inline {
+  height: 100%;
+  width: 100%;
+  .inline-iframe {
+    width: 100%;
+    height: 100%;
+  }
+}
+</style>
```
Comment (Hongbusi) [inline src/views/page/inline/index.vue:20]: 使用 unocss

File: src/router/routes/modules/page.ts [hunk 2]
```diff
@@ -11,6 +11,12 @@ const route: RouteRecordRaw = {
...
       component: () => import('~/views/page/not-found/index.vue')
+    },
+    {
+      path: 'inline',
+      name: 'inline',
+      component: () => import('~/views/page/inline/index.vue'),
+      props: true
     }
   ]
 }
```
Comment (Hongbusi) [inline src/router/routes/modules/page.ts:22]: 可以新增一个 `views/docs` 目录来做文档页面，`page` 一开始的目的是存放错误页面，后续可能会改动。

File: src/views/page/inline/index.vue (hunk 1 above)
Comment (wang01h2) [reply src/views/page/inline/index.vue]: 改好提交啦

File: src/router/routes/modules/page.ts (hunk 2 above)
Comment (wang01h2) [reply src/router/routes/modules/page.ts:22]: 改好提交啦

**Comment by Hongbusi** [comment]
@alexzhang1030 需要你来 review 一下，看看导航需要怎么修改～

**Comment by alexzhang1030** [comment]
> @alexzhang1030 需要你来 review 一下，看看导航需要怎么修改～

待会看一下

**Pull request closed without merging**